HARK/simulation.py

```diff
@@ -33,19 +33,13 @@ def drawMeanOneLognormal(N, sigma=1.0, seed=0):
     # Set up the RNG
     RNG = np.random.RandomState(seed)
```

Comment:
This isn't needed in this function as the seed is passed on to `drawLognormal`.

Reply:
good call